src/combinatorics/simple_graph/basic.lean

```diff
@@ -363,6 +364,35 @@ begin
   exact ⟨v, by simp, finset.subset_univ _⟩,
 end
 
+lemma card_common_neighbors_le_degree (v w : V) :
+  fintype.card (G.common_neighbors v w) ≤ G.degree v :=
+begin
+  rw [←card_neighbor_set_eq_degree],
+  exact set.card_le_of_subset (set.inter_subset_left _ _),
+end
+
+lemma card_common_neighbors_lt_card_verts (v w : V) :
```

Comment:
Is `card_verts` the usual way to refer to the rhs here? I expect `fintype.card V` is a quantity we'll refer to a lot in mathlib graph theory so would be nice to standardise the name

Reply:
So far we have `degree_lt_card_verts`